appveyor.yml

```diff
@@ -0,0 +1,17 @@
+version: 1.0.{build}
+environment:
+  nodejs_version: "4.2.2"
+os:
+  - Visual Studio 2015
+  - Visual Studio 2013
+install:
+  - ps: Install-Product node $env:nodejs_version x64
+  - npm install -g node-gyp
+  - npm install -g mocha
+  - curl -fLsS -o 'gtk.zip' 'http://ftp.gnome.org/pub/GNOME/binaries/win64/gtk+/2.22/gtk+-bundle_2.22.1-20101229_win64.zip'
+  - 7z x gtk.zip -oGTK > nul
+  - mv GTK/ C:/
+  - npm install
+build: off
+test_script:
+- cmd: mocha test/*.test.js
```

Comment:
`npm test` [runs pretest, then test](https://docs.npmjs.com/misc/scripts). pretest currently builds. `npm install` also builds, so it was doing a redundant build. Do we need pretest to build? I agree it would be nicer to just have `npm test`.